hack/test-cmd.sh

```diff
@@ -98,6 +98,17 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+osc get templates
+osc create -f examples/sample-app/application-template-dockerbuild.json
```

Comment:
this is going to be confusing behavior for users i think.  (that you are creating a template object, not creating the things defined in the template).  can't immediately think of a way to fix it that retains consistency with the create verb, but it bothers me so i figured i'd note it in case someone else has a thought.